cfpb/consumerfinance.gov
# ROUT: remove `group sans border-top-2` wrapper

Rural or underserved tool has a wrapper div for the about section that has the classes `group sans border-top-2`. I could not find any styles associated with these classes. 


## Removals

- ROUT: remove `group sans border-top-2` wrapper div.

## How to test this PR

1. Visit https://www.consumerfinance.gov/rural-or-underserved-tool/ and manually remove `group sans border-top-2` and see there is no effect. Visit http://localhost:8000/rural-or-underserved-tool/ and see that the removal of the wrapper div has no effect.
